src/utils.coffee

```diff
@@ -177,10 +177,29 @@ exports.resinLogControl = (val) ->
 	logger.disableLogPublishing(val == 'false')
 	console.log('Logs enabled: ' + val)
 
+gosuperRequest = (method, endpoint, options = {}, callback) ->
+	if _.isFunction(options)
+		callback = options
+		options = {}
+	options.method = method
+	options.url = config.gosuperAddress + endpoint
+	options.headers ?= {}
+	options.headers['Host'] = ''
```

Comment:
Would it be better/nicer to use [request.defaults](https://github.com/request/request#requestdefaultsoptions) for this?